contents/docs/cdp/destinations/webhook.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Send PostHog event data to any webhook
+templateId:
+    - template-webhook
+---
+
+import Requirements from "../_snippets/requirements.mdx"
+import FeedbackQuestions from "../_snippets/feedback-questions.mdx"
+import PostHogMaintained from "../_snippets/posthog-maintained.mdx"
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+With webhook destinations, you can send event data from PostHog to any HTTP endpoint, whether it's to your own backend, an internal system, or a third-party platform.
+
+<Requirements />
+
+## Setting up destination webhooks
+
+### 1. Create a new destination
+
+In PostHog, navigate to the [data pipelines](https://us.posthog.com/pipeline/overview) section in the sidebar. Click **+ New** > **Destination** in the top-right corner. Search for "Webhook" then click **+ Create**.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/webhook_destination_cd495788fc.png"
+  alt="Create new destination"
+  classes="rounded"
+/>
+
+### 2. Configure the webhook
+
+On the configuration page, enter a **Webhook URL**. This is the HTTP endpoint that will receive the event data from PostHog. By default, it sends a `POST` request with a templated JSON body.
+
+Click **Create & Enable**.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/webhook_url_config_5900a5a9b7.png"
+  alt="Configure webhook"
+  classes="rounded"
+/>
+
+### 3. Test the webhook
+
+Now that the webhook is enabled, let's see it in action. In the **Testing** section of the configuration page, click **Start testing** > **Test function** to send an example event. 
+
+You should see event data flow into the webhook URL. 
```

Comment:
Do you have a recommendation for a void server or where to send the webhook to? webhook.site?